docs/images/app/sync-state-diagram.mermaid

```diff
@@ -0,0 +1,35 @@
+stateDiagram
+
+[*] --> LoggedIn
+[*] --> Offline
+
+LoggedIn --> Offline
+state "logged in" as LoggedIn {
+    [*] --> SteadyState
+
+    Note right of SteadyState
+        No UI indicator
+    end note
+    state "steady state" as SteadyState {
+        Idle --> SyncingMetadata: every 15 sec
```

Comment:
@cfm I took the liberty to remove the ambiguity here for now; if we settle on something else, we can always update it.